Makefile

```diff
@@ -50,3 +50,11 @@ clear-cache:
 	docker rm -vf $$(docker ps -a -q) 2>/dev/null ; \
 	docker rmi -f $$(docker images -a -q) 2>/dev/null ; \
 	docker volume rm $$(docker volume ls -q) 2>/dev/null ; true
+
+stop-trial-runners:
+	docker stop $$(docker ps -f "name=fuzzbench-r-*" -q) 2>/dev/null ; \
```

Comment:
i think `docker stop` sends SIGTERM. is that always enough, or we might need `docker kill` sometimes, that sends SIGKILL?

Reply:
Right. SIGTERM is not always enough and SIGKILL is more powerful and suitable here if we intend to remove those containers completely.
Let's combine `docker stop` and `docker rm` into `docker rm --force` as suggested in another comment. Thanks.